internal/display/apps.go

```diff
@@ -238,6 +238,10 @@ func (r *Renderer) ApplicationCreate(client *management.Client, revealSecrets bo
 		ansi.Faint("Hint:"),
 		client.GetClientID(),
 	)
+	r.Infof("%s You might wanna try 'auth0 quickstarts download --client-id %s'",
```

Comment:
Now it's `auth0 quickstarts download <client-id>` please check for other outdated instances.

Reply:
Couldn't find any other outdated refs for quickstarts download 👍

Reply:
Thanks!